app/components/contributors-widget/template.hbs

```diff
@@ -1,17 +1,23 @@
 <table style="width: 100%;"class="list-widget widget-body">
```

Comment:
List-widget and contributor widget were going to be just one right? Are we keeping contributors-widget?